### PR TITLE
Gattling

### PR DESCRIPTION
**Pull request by mrcmatuszak**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -128,4 +141,4 @@ lazy val root = (project in file("."))
   ).aggregate(`tck`, `tck-runner`, `akka-http-mock`, `akka-http-mock-server`)
```
Comment (lustefaniak) [inline build.sbt]: add performance to aggregate, otherwise it is not compiled and tested at all

File: perf/src/test/scala/domofon/scenario.scala [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+  val `update-contact-scenario` = scenario("Update Contact Scenario")
+    .feed(contactFeeder)
+    .exec(
+      createContact,
+      addDeputy,
+      removeDeputy,
+      changeImportance
+    )
+  private def i(v: => String): String = v.replaceAllLiterally("@", "$")
```
Comment (lustefaniak) [inline perf/src/test/scala/domofon/scenario.scala]: Why is it needed?

Comment (mrcmatuszak) [inline perf/src/test/scala/domofon/scenario.scala]: That's a little odd, I know.
To access _feeder_ variables you have to use interpolation-like syntax -  ${contact_id}. But the compiler complains that this variable does not exist. This is a sort of a workaround.

Comment (lustefaniak) [inline perf/src/test/scala/domofon/scenario.scala]: Ok, I suspected that ;)

File: build.sbt (hunk 1 above)
Comment (mrcmatuszak) [inline build.sbt]: I added alias to ';compile;performance/test:compile' which is run by Travis

**Pull request merged**